NuRadioReco/detector/RNO_G/RNO_single_station.json

```diff
@@ -0,0 +1,380 @@
+{
+  "channels": {
+    "0": {
+      "ant_comment": "phased array",
+      "amp_type": "iglu",
+      "adc_sampling_frequency": 2.4,
+      "adc_n_samples": 1024,
+      "ant_orientation_phi": 0.0,
+      "ant_orientation_theta": 0.0,
+      "ant_position_x": 0.0,
+      "ant_position_y": 20.0,
+      "ant_position_z": -97.0,
+      "ant_rotation_phi": 90.0,
+      "ant_rotation_theta": 90.0,
+      "ant_type": "vpol_4inch_center",
+      "cab_time_delay": 0.0,
+      "channel_id": 0,
+      "station_id": 11
+    },
+    "1": {
+      "ant_comment": "phased array",
+      "amp_type": "iglu",
+      "ant_orientation_phi": 0.0,
+      "ant_orientation_theta": 0.0,
+      "ant_position_x": 0.0,
+      "ant_position_y": 20.0,
+      "ant_position_z": -96.0,
+      "ant_rotation_phi": 90.0,
+      "ant_rotation_theta": 90.0,
+      "ant_type": "vpol_4inch_center",
+      "channel_id": 1,
+      "station_id": 11
+    },
+    "2": {
+      "ant_comment": "phased array",
+      "amp_type": "iglu",
+      "ant_orientation_phi": 0.0,
+      "ant_orientation_theta": 0.0,
+      "ant_position_x": 0.0,
+      "ant_position_y": 20.0,
+      "ant_position_z": -95.0,
+      "ant_rotation_phi": 90.0,
+      "ant_rotation_theta": 90.0,
+      "ant_type": "vpol_4inch_center",
+      "channel_id": 2,
+      "station_id": 11
+    },
+    "3": {
+      "ant_comment": "phased array",
+      "amp_type": "iglu",
+      "ant_orientation_phi": 0.0,
+      "ant_orientation_theta": 0.0,
+      "ant_position_x": 0.0,
+      "ant_position_y": 20.0,
+      "ant_position_z": -94.0,
+      "ant_rotation_phi": 90.0,
+      "ant_rotation_theta": 90.0,
+      "ant_type": "vpol_4inch_center",
+      "channel_id": 3,
+      "station_id": 11
+    },
+    "4": {
+      "ant_comment": "power string Hpol",
+      "amp_type": "iglu",
+      "ant_orientation_phi": 0.0,
+      "ant_orientation_theta": 0.0,
+      "ant_position_x": 0.0,
+      "ant_position_y": 20.0,
+      "ant_position_z": -93.0,
+      "ant_rotation_phi": 90.0,
+      "ant_rotation_theta": 90.0,
+      "ant_type": "RNOG_quadslot_finalMatch",
```

Comment:
needs to be renamed to something more meaningful. I suggest `RNOG_quadslot_v3_air_rescaled_to_n1.74`